lib/test/clojure_lsp/features/completion_test.clj

```diff
@@ -229,7 +229,7 @@
       [{:label "comment"
         :detail "clojure.core/comment"
         :insert-text "(comment\n  ${0:body}\n  )"
-        :kind :reference
+        :kind :snippet
```

Comment:
Why this was not failing before?

Reply:
hmmm I thought it was going to be an additional field not a change on the data structure, but `kind` into `:snippet-kind` feels like restructure no? I'm worried it might break existing setups but anyways whatever @ericdallo see appropriate.

Reply:
Yeah, a additional field outside of data seems wrong and not following the spec, the spec has the data field for client route the data to server again, although I'm okay adding a field to the data, but yes, it looks to me clojure-lsp should return the snippet as :kind indeed

Reply:
@tami5 It would not break the current implementation in that sense, since `:kind` still exists and still returns a completion-kind. The "enriched" information would move to `:data {:snippet-kind "kind"}` for anyone who wants to display that instead, which is "only" vim I think.

But yes @ericdallo one of the IDEs will have to do extra work since their handling of completionItem display is different.